kubernetes-deploy.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency("googleauth", "~> 0.8.0")
   spec.add_dependency("ejson", "~> 1.0")
   spec.add_dependency("colorize", "~> 0.8")
-  spec.add_dependency("statsd-instrument", '~> 2.3', '>= 2.3.2')
```

Comment:
I think we should pin this as `>= 2.3.2` and `< 2.5.0`?

Reply:
that's technically correct, but only allows for 2.4.0 which i don't think we'll bother with anyways, we'll need to update this for #576 anyways